service/db/src/main/java/gov/va/vro/service/db/SaveToDbServiceImpl.java

```diff
@@ -179,6 +183,35 @@ public void insertEvidenceSummaryDocument(GeneratePdfPayload request, String doc
     claimRepository.save(claim);
   }
 
+  @Override
+  public void updateEvidenceSummaryDocument(UUID eFolderId, MasAutomatedClaimPayload payload) {
+    var claim =
+        claimRepository.findByVbmsId(String.valueOf(payload.getClaimDetail().getBenefitClaimId()));
+    if (claim.isEmpty()) {
+      log.error("Could not find claim with vbmsId. Could not attach eFolder ID.");
```

Comment:
if we are allowed to log vbms id here i would include that. as  if we're storing it it's not PII.